packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -17,6 +19,29 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
   const { t } = useTranslation();
   const { api } = useApi();
   const { nomsActive, nomsChilled, nomsInactive, nomsOver, nomsWaiting } = useInactives(stashId, nominating);
+  const sessionInfo = useCall<DeriveSessionProgress>(api.query.staking && api.derive.session?.progress);
+  const eraExposure = useCall<DeriveEraExposure>(api.derive.staking?.eraExposure, [sessionInfo?.activeEra]);
+  const nomBalanceMap: Record<string, BN> = useMemo(() => {
+    const res: Record<string, BN> = {};
+
+    if (nomsActive && eraExposure) {
+      // for every active nominee
+      nomsActive.forEach((nom) => {
+        // cycle through its nominator to find our current stash
+        eraExposure.validators?.[nom].others.some((o) => {
+          if (o.who.toString() === stashId) {
```

Comment:
```suggestion
          if (o.who.eq(stashId)) {
```

Reply:
TIL again, didn't think that would work with BN and string!

Reply:
who is AccountId, all the polkadot-js Codec stuff has .eq on them. (And in this case can compare hex, Uint8array, other AccountId and ss58)